dtfs-central-api/src/helpers/execute-with-sql-transaction.ts

```diff
@@ -71,6 +71,8 @@ export const executeWithSqlTransaction = async <ReturnValue>(functionToExecute:
 
     return result;
   } catch (error) {
+    console.error('Error thrown within SQL transaction: %s', error);
```

Comment:
Picky - we seem to be using `%o` in the majority of places where we're taking this approach

```suggestion
    console.error('Error thrown within SQL transaction: %o', error);
```